thirdparty/stb_rect_pack/stb_rect_pack.h

```diff
@@ -0,0 +1,629 @@
+// stb_rect_pack.h - v1.00 - public domain - rectangle packing
```

Comment:
This can go in `thirdparty/misc/` like other `stb` headers, and like done in `master`.

Also make sure to document it in `thirdparty/README.md` (under "Misc / Modules", you can copy from `stb_vorbis.c` and adjust).